examples/DateAxisItem_QtDesigner.py

```diff
@@ -34,8 +37,19 @@ def __init__(self):
         self.plotWidget.setAxisItems({'bottom': pg.DateAxisItem()})
         self.plotWidget.showGrid(x=True, y=True)
 
+# Qt docs have the following warning regarding the following function:
+#   "To ensure that the application's style is set correctly, it is best
+#   to call this function before the QApplication constructor, if possible"
+QtWidgets.QApplication.setStyle('Fusion')
 app = QtWidgets.QApplication(sys.argv)
-app.setStyle(QtWidgets.QStyleFactory.create('Fusion'))
+if 0:
```

Comment:
Should this be an if statement something like `if pg.Qt.QT_LIB != "PyQt6":` ?

Reply:
setStyle has been moved to be called before the constructor of QApplication, valid for all bindings, and recommended by the Qt docs.

I.e. the contents of "if 0:" are not needed for any binding. It's only left behind as documentation for the PyQt6 bug. But I see that this coding style triggers a static analyser warning about unreachable code, so maybe I should fully comment it out.

Reply:
Should the call to `setStyle` be made in `mkQApp`?

Reply:
So I could:
1) add a style keyword to mkQApp
Or
2) don't set the style altogether

Because in the whole library, there is only this one usage of setStyle and only inside a small example. setStyle doesn't really add any functional value.

Reply:
I'm good with not setting the style altogether (assuming that this example would still work w/o the style being set).  `mkQApp` contains contents for the pyqtgraph examples to run the way they should, but we have no control over the `QApplication` instantiation end-users create.